stanford-sw/src/edu/stanford/StanfordIndexer.java

```diff
@@ -1875,6 +1893,40 @@ private void setFundFacet(final Record record) {
   }
 }
 
+/**
+ * returns locations based upon the data in the 852 subfield c or 999 subfield t
+ * @param record a marc4j Record object
+ */
+ public Set<String> getLocationFacet(final Record record)
+ {
+  return locationFacet;
+ }
+
+/**
+* set locations based upon the value in the 852 subfield c or 999 subfield t
+* "Art Locked Stacks"
+* @param record a marc4j Record object
+*/
+private void setLocationFacet(final Record record) {
+
+  for (Item item : itemSet) {
+    if (item.hasArtLockedLoc()) {
+      locationFacet.add("Art Locked Stacks");
+    }
+  }
+
+  List<VariableField> list852 = record.getVariableFields("852");
+  for (VariableField vf : list852)
+  {
+    DataField df = (DataField) vf;
+    String subC = MarcUtils.getSubfieldData(df, 'c');
+    if (ART_LOCKED_LOCS.contains(subC)){
+      locationFacet.add("Art Locked Stacks");
```

Comment:
works for me. mebbe I'll add a comment - looks like i have to tweak .travis.yml in any case.